oci/config/convert/runtime.go

```diff
@@ -91,17 +88,23 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image, manif
 		g.AddProcessEnv(name, value)
 	}
 
-	// We don't append to g.Spec().Process.Args because the default is non-zero.
-	// FIXME: Should we make this instead only append to the pre-existing args
-	//        if Entrypoint != ""? I'm not really sure (Docker doesn't).
 	args := []string{}
 	args = append(args, image.Config.Entrypoint...)
 	args = append(args, image.Config.Cmd...)
-	if len(args) == 0 {
-		args = []string{"sh"}
+	if len(args) > 0 {
+		g.SetProcessArgs(args)
 	}
-	g.SetProcessArgs(args)
 
+	// Set annotations fields
+	for key, value := range image.Config.Labels {
+		g.AddAnnotation(key, value)
+	}
+	g.AddAnnotation("author", image.Author)
+	g.AddAnnotation("created", image.Created.Format(gspec.ISO8601))
+	// FIXME: currently not supported! Uncomment when moving to image-spec rc5.
+	// g.AddAnnotation("StopSignal", image.Config.StopSignal)
```

Comment:
All of these annotations key names aren't right (you need to use the full `org.opencontainers.<blah>.<blah>` namespacing as per the conversion doc. From the doc:

> 2. The value of this field MUST be set as the value of `org.opencontainers.image.author` in `annotations`.
> 3. The value of this field MUST be set as the value of `org.opencontainers.image.created` in `annotations`.
> 4. The value of this field MUST be set as the value of `org.opencontainers.image.stopSignal` in `annotations`.